Authentication-Results: mail-a.sr.ht; dkim=pass header.d=xry111.site header.i=@xry111.site
Received: from xry111.site (xry111.site [89.208.246.23])
	by mail-a.sr.ht (Postfix) with ESMTPS id BDE3A20506
	for <~kennylevinsen/greetd-devel@lists.sr.ht>; Wed, 08 Apr 2026 02:29:07 +0000 (UTC)
DKIM-Signature: v=1; a=rsa-sha256; c=relaxed/relaxed; d=xry111.site;
	s=default; t=1775615345;
	bh=XlJQAIi3kb901xOeSJ2pdOCcoxYhrRDlu39sHaJWrok=;
	h=From:To:Cc:Subject:Date:From;
	b=iVG1g1PWIbdvI3G6l2HVBCugxvjJMR1Q4IMQiIi8UY7CqNNz+zAhlVHh90Vl0Vp73
	 7NSYKURStKQFadMDVMLBtQACsgygY00vYsplXmzlSZ9bkN45qAbju6BfNeP0NGVlIG
	 zYIDU4ohYurP24Nk7N/seoN/ka0OtJhguEpXOwHs=
Received: from stargazer (unknown [IPv6:2409:8a4c:e10:180:5633:9efb:acb8:d769])
	(using TLSv1.3 with cipher TLS_AES_256_GCM_SHA384 (256/256 bits)
	 key-exchange X25519 server-signature ECDSA (secp384r1) server-digest SHA384)
	(Client did not present a certificate)
	(Authenticated sender: xry111@xry111.site)
	by xry111.site (Postfix) with ESMTPSA id 40C5E65992;
	Tue,  7 Apr 2026 22:29:03 -0400 (EDT)
From: Xi Ruoyao <xry111@xry111.site>
To: ~kennylevinsen/greetd-devel@lists.sr.ht
Cc: Xi Ruoyao <xry111@xry111.site>
Subject: [PATCH] systemd: add Conflicts with kmsconvt
Date: Wed,  8 Apr 2026 10:28:50 +0800
Message-ID: <20260408022850.176889-1-xry111@xry111.site>
X-Mailer: git-send-email 2.53.0
MIME-Version: 1.0
Content-Transfer-Encoding: 8bit

kmsconvt is a replacement for getty, so it will also conflict with gdm.
Add Conflicts= for it like what plasmalogin and gdm have done.

Signed-off-by: Xi Ruoyao <xry111@xry111.site>
---
 greetd.service | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/greetd.service b/greetd.service
index 4fad1b0..03b20f2 100755
--- a/greetd.service
+++ b/greetd.service
@@ -1,8 +1,8 @@
 [Unit]
 Description=Greeter daemon
 After=systemd-user-sessions.service plymouth-quit-wait.service
-After=getty@tty1.service
-Conflicts=getty@tty1.service
+After=getty@tty1.service kmsconvt@tty1.service
+Conflicts=getty@tty1.service kmsconvt@tty1.service
 
 [Service]
 Type=simple
-- 
2.53.0

